sdk/tests/js/js-test-pre.js

```diff
@@ -214,6 +214,9 @@ var TestFailedException = function (message) {
 
 function testPassed(msg)
 {
+    if (_currentTestName)
+      msg = _currentTestName + ': ' + msg;
```

Comment:
Perhaps you could simplify the callers by writing something like:

  if (!msg)
    msg = 'Passed'
  if (_currentTestName)
    ...